hrpc/mutate.go

```diff
@@ -371,61 +371,40 @@ func (m *Mutate) valuesToProto(ts *uint64) []*pb.MutationProto_ColumnValue {
 const nSlicesPerCellblock = 7
 const lengthsAndTsSize = 4 + 4 + 4 + 2 + 1 + 8 + 1
 
-func toCellblock(row, family, qualifier, value []byte, ts uint64, typ byte) ([][]byte, uint32) {
-	var cellblock [][]byte
-	if len(value) != 0 {
-		cellblock = make([][]byte, nSlicesPerCellblock)
-	} else {
-		cellblock = make([][]byte, nSlicesPerCellblock-1)
-	}
-	lengthsAndTs := make([]byte, lengthsAndTsSize)
-
+func toCellblock(row, family, qualifier, value []byte, ts uint64, typ byte) []byte {
 	// keyvaluelength
 	keylength := 2 /* row length */ + len(row) + 1 /* columnfamily length */ +
 		len(family) + len(qualifier) + 8 /* timestamp length */ + 1 /* key type length */
 	valuelength := len(value)
 
 	keyvaluelength := 8 + uint32(keylength+valuelength)
+	cellblock := make([]byte, keyvaluelength+4)
 
-	// keyvaluelength
```

Comment:
Would be nice to keep the comments for clarity

Reply:
I had felt the code was more clear now so didn't need the comments. I'll take another look.

Reply:
I added a comment with the the layout of a cellblock. I didn't think the line-by-line comments of the form `copy(cellbock[i:], foo) // adding foo` were very useful. Let me know what you think.